src/Sentry.Unity/Integrations/IApplication.cs

```diff
@@ -25,6 +25,7 @@ public sealed class ApplicationAdapter : IApplication
 
         private ApplicationAdapter()
         {
+            // TODO why not logMessageReceivedThreaded? We're currently ignoring errors from other than the main thread.
```

Comment:
I believe we receive events from other threads but they get queued to be pushed in the UI thread, isn't that the case? We used to have that but it had issues with processing events from the background thread because the SDK used APIs that required the UI thread. It might have changed now (I believe we already have checks "don't retrieve this data if NOT running in the UI thread) so we might be able to move back to `logMessageReceivedThreaded`.

If we confirm that ALL events go through `logMessageReceived` the trade off becomes "cause less overhead by capturing stuff on the background when that's possible but losing additional context that requires the UI thread"

Reply:
I've tested it by running `Task.Run(() => Debug.LogError("Debug.LogError() in a background thread"));` and it's not forwarded, while the plain `Debug.LogError()` is. I'd say we should address this in https://github.com/getsentry/sentry-unity/issues/471 & add buttons for the test cases (or maybe a global checkbox that causes all buttons to work on background). Leaving this conversation unresolved and part of the main issue.

Reply:
resolved by https://github.com/getsentry/sentry-unity/pull/673